AnimalsTest/.editorconfig

```diff
@@ -0,0 +1,143 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
```

Comment:
editorconfig should be placed in the root of the repo, so it applies to all code files. Additional editorconfig rules may be applied to separate projects (disable/enable specific rules for that directory on top of main rules).